src/status_im/constants.cljs

```diff
@@ -491,3 +491,7 @@
 
 (def ^:const pre-login-log-level-key "pre-login-log-level")
 (def ^:const report-email "error-reports@status.im")
+
+;; Community help links
+(def ^:const create-community-help-url
+  "https://status.app/help/communities/create-a-status-community#create-a-status-community")
```

Comment:
Maybe we should organize all status help links in a dedicates section/file so that we can track and update them easily. wdyt? @ulisesmac @ilmotta

![image](https://github.com/user-attachments/assets/240603df-51ad-4d5b-b7b2-aa35a060d7b7)

Reply:
Nice idea 👍🏼

Reply:
For community related links, I added the section. I still feel there isn't enough (yet) to properly define a pattern, it's a good idea, but IMO let's wait a bit more so we can create a better abstraction. :smile:  wdyt?